### DISCUSSION TITLE
hubserving部署structure_table效果比通用表格识别差不少

### DISCUSSION THREAD

**Discussion by Tongjilibo** [PaddlePaddle/PaddleOCR#12261] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Linux
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

使用hubserving部署structure_table接口，用的模型是`ch_ppstructure_mobile_v2.0_SLANet`，调用后发现对于同一张表格图片识别效果一般，但是通用[表格识别demo](https://aistudio.baidu.com/community/app/91661/webUI)能识别正确，这是因为啥呢？

**Comment by TingquanGao** [comment]
做服务化部署的话，建议使用PaddleX导出服务化部署包，或是了解一下PaddleServing。

**Comment by Tongjilibo** [comment]
> 做服务化部署的话，建议使用PaddleX导出服务化部署包，或是了解一下PaddleServing。

问下这几种部署方式在效果上会有区别嘛？

**Comment by tjuzdy** [comment]
我是做成了docker镜像，用hub install /PaddleOCR/deploy/hubserving/structure_table/安装的，启动后调8866端口容器就挂了，你这边是如何调用测试的呢

**Comment by Tongjilibo** [comment]
> 我是做成了docker镜像，用hub install /PaddleOCR/deploy/hubserving/structure_table/安装的，启动后调8866端口容器就挂了，你这边是如何调用测试的呢

我没用docker直接跑的，可以看下知乎上写的一个[简单介绍](https://zhuanlan.zhihu.com/p/673410438?)，只是效果不如paddle给的[示例](https://aistudio.baidu.com/community/app/91661/webUI)好
